src/main/java/thePackmaster/packs/SiegePack.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.packs;
+
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cards.siegepack.*;
+
+import java.util.ArrayList;
+
+public class SiegePack extends AbstractCardPack {
+    public static final String ID = SpireAnniversary5Mod.makeID("SiegePack");
+    private static final UIStrings UI_STRINGS = CardCrawlGame.languagePack.getUIString(ID);
+    public static final String NAME = UI_STRINGS.TEXT[0];
+    public static final String DESC = UI_STRINGS.TEXT[1];
+    public static final String AUTHOR = UI_STRINGS.TEXT[2];
+
+    public SiegePack() {
+        super(ID, NAME, DESC, AUTHOR, new PackSummary(3, 4, 3, 2, 2, PackSummary.Tags.Attacks, PackSummary.Tags.Debuffs, PackSummary.Tags.Powers));
```

Comment:
```suggestion
        super(ID, NAME, DESC, AUTHOR, new PackSummary(3, 4, 3, 2, 2, PackSummary.Tags.Attacks));
```
Those tags mean synergy with, not applicable